.openpublishing.redirection.json

```diff
@@ -1125,6 +1125,30 @@
         {
             "source_path": "docs/csharp/programming-guide/strings/how-to-compare-strings.md",
             "redirect_url": "/dotnet/csharp/how-to/compare-strings"
+        },
+        {
+            "source_path": "docs/fsharp/tutorials/type-providers/accessing-a-sql-database-entities.md",
+            "redirect_url": "docs/fsharp/tutorials/type-providers/index"
+        },
+        {
+            "source_path": "docs/fsharp/tutorials/type-providers/accessing-a-sql-database.md",
+            "redirect_url": "docs/fsharp/tutorials/type-providers/index"
+        },
+        {
+            "source_path": "docs/fsharp/tutorials/type-providers/accessing-a-web-service.md",
+            "redirect_url": "docs/fsharp/tutorials/type-providers/index"
+        },
+        {
+            "source_path": "docs/fsharp/tutorials/type-providers/accessing-an-odata-service.md",
+            "redirect_url": "docs/fsharp/tutorials/type-providers/index"
+        },
+        {
+            "source_path": "docs/fsharp/tutorials/type-providers/generating-fsharp-types-from-dbml.md",
+            "redirect_url": "docs/fsharp/tutorials/type-providers/index"
+        },
+        {
+            "source_path": "docs/fsharp/tutorials/type-providers/generating-fsharp-types-from-edmx.md",
+            "redirect_url": "docs/fsharp/tutorials/type-providers/index"
```

Comment:
@cartermp `redirect_url` are not correct: *docs* substring should be replaced with *dotnet*

Reply:
Good catch @pkulikov. I'll make a new PR to fix that.